packages/mdc-typography/_mixins.scss

```diff
@@ -35,6 +35,10 @@
   font-weight: #{map-get($style-props, font-weight)};
   letter-spacing: map-get($style-props, letter-spacing);
   line-height: map-get($style-props, line-height);
+
+  @if $style == "button" {
```

Comment:
can we just set text-decoration: none on all of the typography styles? We never really want decoration, button isn't special in this way.

And can we put text-transform: none for the other typography styles? Instead of having this if-statement just for button.